### ISSUE TITLE
Allow moving the color picker panel

### ISSUE THREAD

**Issue by lostminds** [godotengine/godot-proposals#5764]
### Describe the project you are working on

Since this is so basic I don't think the context will matter much, but I've run into the issue editing material colors for 3D model material like in the example below.

### Describe the problem or limitation you are having in your project

When clicking a color parameter to edit it the Color Picker in Godot 4.0.beta4 pops up on top positioned at the top of the Inspector panel. This may even be a bug, but I've found no way of clicking and dragging to move the panel, and since it's so big the positioning is a problem since the panel obscures the Material preview. So as you change or set the color you can't see the impact it will have on the material in the preview. 

https://user-images.githubusercontent.com/17763524/201349169-ab3a13b7-9e05-47b0-8255-48ec51622d72.mov

This is just a very specific case, but the more general issue here is that it seems you can't move the color picker panel. And since it's bound to be positioned poorly in some cases this is a usability problem since you then need to close and open the color picker over and over.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Simply allow clicking and dragging the color picker panel to move it. This would allow users to move it out of the way if it's placed inconveniently, and make it less sensitive where it's first shown.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

I assume there is already code in place to for dragging and moving popup panels and such, so it's just a matter of enabling these for the Color Picker panel. 

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No, since it's a usability issue with an editor UI component I don't think so

### Is there a reason why this should be core and not an add-on in the asset library?

it should be basic functionality of this standard editor component

**Comment by Calinou** [comment]
Which area should allow dragging the color picker? Should it be any area that doesn't serve any clickable purpose within the popup?

**Comment by lostminds** [comment]
> 

Yes, anywhere in the panel background sounds good as it is now. If there was a title bar at the top (which might be a good ideas for other reasons, like adding a close button or edited parameter title) the panel title bar would also be a good candidate for drag-moving the panel.
